Source/loadsave.cpp

```diff
@@ -2539,10 +2539,7 @@ void SaveGameData(SaveWriter &saveWriter)
 			for (int i = 0; i < MAXDUNX; i++) // NOLINT(modernize-loop-convert)
 				file.WriteLE<int8_t>(dObject[i][j]);
 		}
-		for (int j = 0; j < MAXDUNY; j++) {
-			for (int i = 0; i < MAXDUNX; i++)        // NOLINT(modernize-loop-convert)
-				file.WriteLE<uint8_t>(dLight[i][j]); // BUGFIX: dLight got saved already
-		}

```

Comment:
Does vanilla read this one or the other one? If we get it wrong vanilla will be pitch black when loading saves from devx

Reply:
Vanilla reads the first one if it's loading town, and the second one if it's loading dungeon. I guess it would be safer to just leave it how it is.

Reply:
Vanilla doesn't care about lighting in town as it uses a separate render for town that doesn't support as many features as the one for the dungeon. Meaning you can remove the first one, but not the second.